config/locales/thematic_area/target_11_dashboard/en.yml

```diff
@@ -15,20 +15,25 @@ en:
         -
           id: coverage
           title: Coverage
-          text: "Proportion of the world's terrestrial or marine area covered by protected areas <a href='/thematical-areas/oecms' title='View OECMs' target='_blank'>and OECMs</a>. Statistics updated monthly. Source: <a href='https://www.protectedplanet.net/' title='View source' target'_blank'>WDPA and WD-OECM</a> (UNEP-WCMC & IUCN)"
+          text: &coverage  ! "Proportion of %{geo_type} terrestrial or marine area covered by protected areas <a href='/thematical-areas/oecms' title='View OECMs' target='_blank'>and OECMs</a>. Statistics updated monthly. Source: <a href='https://www.protectedplanet.net/' title='View source' target'_blank'>WDPA and WD-OECM</a> (UNEP-WCMC & IUCN)"
```

Comment:
It is a YAML anchor which allows me to refer and reuse this value elsewhere in the script, which I can access using an alias, which is what the keys with * in front of them are. I used it because I still wanted the tooltips as an array, but wanted a way of referencing them to allow me to interpolate strings into them. The ! is a non-specific tag which means that the value of the YAML node which be resolved to its 'native' data structure